pallets/session-ext/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "pallet-session-ext"
```

Comment:
`SessionExt` is a pretty terrible name for a pallet, I would suggest going for something (perhaps) longer, but more descriptive. Maybe `AlephAuthoritiesSession` or something?

Reply:
I have no idea why but I have read that as AlephSessionManager...

Reply:
will do the renaming to AlephAuthorithiesSession :|

Reply:
done, as discussec rename to CommiteeManagement